### ISSUE TITLE
[Python] Spec definition containing brackets generates model with invalid filename

### ISSUE THREAD

**Issue by cbdelavenne** [swagger-api/swagger-codegen#2306]
```
{
    "definitions": {
        "SomeDto[AnotherDto]": {
             ...
        }
    }
}
```

When fed through swagger codegen, the definition above generates a file:
`swagger_client/models/some_dto[another_dto].py`

Then `swagger_client/models/__init__.py` can't correctly import the file:
```
from .some_dto[another_dto] import SomeDtoAnotherDto
```

Anyway, all this to say, during generation, brackets in definition/path names should be converted to `_` or empty space when creating files.

Let me know if this isn't clear or if there's a naming convention for swagger spec paths and definitions that I'm not aware of. I couldn't find anything mentioning that in http://swagger.io/specification/

**Comment by wing328** [comment]
@cbdelavenne thanks for reporting the issue, which has been addressed by #2309 

Please pull the latest and let us know if you're still seeing the issue.

We've added some test cases, which will be reused to cover all other languages.

**Comment by cbdelavenne** [comment]
Just had a chance to test it out. It's working properly now. Thanks!
